### DISCUSSION TITLE
Uncaught ReferenceError: Require is not defined

### DISCUSSION THREAD

**Discussion by timtjoe** [vercel/next.js#56859] [Help]
### Summary

Hi

I got this error when I deploy my react app on vercel: **Uncaught ReferenceError: require is not defined**
I changed the nodejs version from 18x to 16x but the error is not resolved.

This is  the error massage logged to the console: 
Uncaught ReferenceError: require is not defined
    at index-08337b32.js:252:2875

Have any encountered this problem recently and how do you solve it?

### Additional information

```js
Uncaught ReferenceError: require is not defined
at index-08337b32.js:252:2875
```

### Example

https://finsinai.vercel.app/

**Comment by RohitPeswani** [comment]
same error how did you solve?

**Reply by wilsonadenuga** [reply]
Do you want to share some code snipette? @RohitPeswani
